examples/adyen-dropin-component/api-types/storeApiTypes.overrides.ts

```diff
@@ -95,7 +96,9 @@ export type Schemas = {
       name?: string;
     };
     /** Selected payment method */
-    paymentMethod?: components["schemas"]["PaymentMethod"]; // TODO: [OpenAPI][SalesChannelContext] paymentMethod field should be defined properly reusing PaymentMethod schema
+    paymentMethod?: components["schemas"]["PaymentMethod"] & {
+      fundingSource: "debit" | "credit" | undefined;
```

Comment:
```suggestion
      fundingSource: components["schemas"]["AdyenPaymentMethod"]["fundingSource"]
```

or maybe even that:
```
      paymentMethod?: components["schemas"]["PaymentMethod"] | components["schemas"]["AdyenPaymentMethod"];
```

it depends how AdyenPaymentMethod is covering PaymentMethod